components/NavLayout/SideBar.js

```diff
@@ -1,12 +1,122 @@
+import Link from 'next/link';
+import { useState } from 'react';
+
+import { styled, useTheme } from '@mui/material/styles';
+import { openedMixin, closedMixin, DrawerHeader, AppBar, Drawer } from '../../components/basecomponents/SideBarStyles.js';
+import { Box, Container, Stack, Avatar } from '@mui/material';
+import { ListItemButton, ListItemIcon, ListItemText } from '@mui/material';
+import { Toolbar, List, CssBaseline, Typography, IconButton } from '@mui/material';
+
+import MenuIcon from '@mui/icons-material/Menu';
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import ClassOutlinedIcon from '@mui/icons-material/ClassOutlined';
+import CalendarMonthOutlinedIcon from '@mui/icons-material/CalendarMonthOutlined';
+import NotificationsOutlinedIcon from '@mui/icons-material/NotificationsOutlined';
+import SearchOutlinedIcon from '@mui/icons-material/SearchOutlined';
+
+
+import defaultProfilePic from '../../utils/constants';
 import styles from '../../utils/styles/NavLayoutStyles/SideBar.module.css';
 
 export default function SideBar({ children, ...props }) {
```

Comment:
Great job separating out the styled base components 😃 . Now when navigate files, we see our component that gets rendered  immediately after our imports which helps with code readability and consequently future debugging.